rel/files/riak-cs-debug

```diff
@@ -455,23 +518,28 @@ if [ 1 -eq $get_logs ]; then
         fi
     fi
 
-    # Lager files
-    lager_paths="`epath 'lager handlers lager_file_backend file' "$riak_epaths" \
-                         | sed -e 's/^"//' -e 's/".*$//'`"
+    # Lager info and error files
+    new_format_lager_files="`epath 'lager handlers lager_file_backend file' "$riak_epaths" | sed -e 's/^"//' -e 's/".*$//'`"
+    if [ -z "$new_format_lager_files" ]; then
+        lager_files="`epath 'lager handlers lager_file_backend' "$riak_epaths" | cut -d' ' -f 1 | sed -e 's/^"//' -e 's/".*$//'`"
+    else
+        lager_files=$new_format_lager_files
+    fi
     lager_num=0
-    for lager_path in $lager_paths; do
+    for lager_path in $lager_files; do
         # Get lager logs if they weren't in platform_log_dir
         if [ -n "$lager_path" ]; then
-            if [ '.' = `echo "$lager_path" | awk 'BEGIN {FS=""} {print $1}'` ]; then
+            if [ '/' != `echo "$lager_path" | cut -c1` ]; then
                 # relative path. prepend base dir
                 lager_path="$cs_base_dir"/"$lager_path"
             fi
 
             lager_file="`echo $lager_path | awk -F/ '{print $NF}'`"
-            lager_dir="`echo $lager_path | awk -F/$lager_file '{print $1}'|sed 's|/./|/|'`"
+            lager_dir="`echo $lager_path | awk -F/$lager_file '{print $1}'`"
             if [ "$cs_log_dir" != "$lager_dir" ]; then
-                if [ -n "`find "${lager_dir}" -maxdepth 1 -mtime -$log_mtime -name "${lager_file}*" -print -quit`" ]; then
-                    mkdir_or_die "${start_dir}"/"${debug_dir}"/logs/lager_"${lager_num}"
+                ls -1 "${lager_dir}" | grep -q "${lager_file}"
+                if [ 0 -eq $? ]; then
+                    mkdir_or_die "${start_dir}"/"${debug_dir}"/logs/lager_${lager_level}
```

Comment:
Although I'm not sure what's wrong, this part looks not to work. Lager's logs are not gathered in the case which console.log and error.log are out of platform_log_dir such as:

```
platform_log_dir = /var/log/riak-cs
log.console.file = /data/log/console.log
log.error.file = /data/log/error.log
```